src/plugin-vite.js

```diff
@@ -12,6 +12,7 @@ import { getNuxtFiles, getVueFiles } from "./utils/node/helper.js";
 import { componentResolver, directiveResolver } from "./utils/node/vuelessResolver.js";
 import { setCustomPropTypes, removeCustomPropTypes } from "./utils/node/dynamicProps.js";
 import { buildWebTypes } from "./utils/node/webTypes.js";
+import { hideStories, showHiddenStories } from "./utils/node/dynamicStories.js";
```

Comment:
hideStories >>> hideHiddenStories,

`hideHiddenStories` and `showHiddenStories` looks more consistent.